tesler-core/src/main/java/io/tesler/core/metahotreload/service/ViewAndViewWidgetUtil.java

```diff
@@ -0,0 +1,93 @@
+/*-
+ * #%L
+ * IO Tesler - Core
+ * %%
+ * Copyright (C) 2018 - 2019 Tesler Contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package io.tesler.core.metahotreload.service;
+
+import static io.tesler.core.metahotreload.util.JsonUtils.serializeOrElseEmptyArr;
+import static java.util.Optional.ofNullable;
+
+import io.tesler.core.metahotreload.dto.ViewSourceDTO;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.tesler.model.ui.entity.View;
+import io.tesler.model.ui.entity.ViewWidgets;
+import io.tesler.model.ui.entity.ViewWidgetsPK;
+import io.tesler.model.ui.entity.Widget;
+import java.util.List;
+import java.util.Map;
+import javax.persistence.EntityManager;
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ViewAndViewWidgetUtil {
+
+	static void process(
```

Comment:
I don't see any usage of "process" methods. Shall we use "process" in client side? Or we need to declare bean in library?

Reply:
Usage was lost in another pr.
fixed